sources/packages/backend/libs/sims-db/src/entities/announcements.model.ts

```diff
@@ -0,0 +1,56 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+
+/**
+ * System Announcements.
+ */
+@Entity({ name: TableNames.Announcements })
+export class Announcements extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Announcement title.
+   */
+  @Column({
+    name: "message_title",
+    nullable: false,
+  })
+  messageTitle: string;
+  /**
+   * Announcement content.
+   */
+  @Column({
+    name: "message",
+    nullable: false,
+  })
+  message: string;
+  /**
+   * Target areas for the announcement.
+   */
+
```

Comment:
Can we please remove the extra line here.

Reply:
updated